packages/core/test/combine-test.ts

```diff
@@ -10,27 +10,29 @@ import { collectEventsFor, makeEvents, makeEventsFromArray } from './helper/test
 
 const sentinel = { value: 'sentinel' }
 
+const args = <Args extends unknown[]>(...args: Args): Args => args
+
 describe('combine', function () {
   it('given one canonical empty stream, should return canonical empty', () => {
-    const s = combine(Array, now(0), empty())
+    const s = combine(args, now(0), empty())
```

Comment:
Interesting.  I'm guessing this is because TS treats the Array constructor as accepting only a homogenous argument list.  Is that why this was necessary?

Reply:
Exactly. Otherwise TS infers the type as `Stream<unknown[]>`